src/components/organisms/Feedbacks/FeedbackCard.jsx

```diff
@@ -1,15 +1,10 @@
 import styled from 'styled-components';
 import Card, { variants as cardVariants } from '../../atoms/Card';
 
-const Photo = styled.div`
-  flex: none;
-
+const Picture = styled.picture`
+  display: block;
   width: 4em;
   height: 4em;
-
-  background-color: ${({ theme }) => theme.colors.secondary};
```

Comment:
You need to leave it. Do not remove

Reply:
@DanilAgafonov  If I leave it, I need to leave border-radius too?

Reply:
See other placed on page where we use images. We usually use `background-color` on `img` tag (not on `picture`). So move it